tests/run-make/amdgpu-kd/rmake.rs

```diff
@@ -0,0 +1,16 @@
+// Checks that the produced object has the .kd symbol exported.
+
+//@ needs-rust-lld
```

Comment:
This also needs a needs-llvm-components: amdgpu I think (forgor exact component name)